src/common/components/loading/loading.scss

```diff
@@ -66,6 +66,13 @@ loading{
     z-index: 2;
   }
 
+  .loading-fixed:first-child {
```

Comment:
The existing options have the loading icon and text centered in the div. Since our form is very long, the user can't see the loading text unless they scroll up. This new style fixes the loading text to the center of the viewport.